$OpenBSD: patch-_metadata,v 1.3 2017/11/04 15:49:24 jeremy Exp $

Don't runtime depend on mini_portile2, it's only need for compiling
the extension.

--- .metadata.orig	Thu Nov  3 16:37:01 2016
+++ .metadata	Thu Nov  3 16:37:43 2016
@@ -21,7 +21,7 @@ dependencies:
     - - "~>"
       - !ruby/object:Gem::Version
         version: 2.3.0
-  type: :runtime
+  type: :development
   prerelease: false
   version_requirements: !ruby/object:Gem::Requirement
     requirements:
